docs/content/envoy/workflows.en.md

```diff
@@ -0,0 +1,214 @@
+    end
+
+    critical
+        Oc ->> Oc: Required KYC and Repair
+    end
+
+    rect rgb(240,186,200)
+        note over O,B: RPC 3
+        O ->> B: Review
+        B ->> O: Pending
+        O ->> Oc: Transaction Pending
+        B ->> Bc: Transaction Review
+    end
+
+    critical
+        Bc->> Bc: Validate fixes meet requirements
+    end
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 4
+        Bc ->> B: Accept Transaction
+        B ->> O: Accepted
+        O ->> B: Accepted
+        B ->> Bc: Transaction Accepted
+        O ->> Oc: Transaction Accepted
+    end
+
+    critical
+        Oc ->> Oc: Perform on chain transaction
+    end
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 5
+        Oc ->> O: Complete Transaction
+        O ->> B: Completed
+        B ->> O: Completed
+        O ->> Oc: Transaction Completed
+        B ->> Bc: Transaction Completed
+    end
+```
+
+## Transaction Rejected and/or Canceled
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant Oc as Originator User
+    participant O as Originator Envoy
+    participant B as Beneficiary Envoy
+    participant Bc as Beneficiary User
+
+    rect rgb(27, 206, 159 )
+        note over O,B: RPC 1
+        Oc ->> O: Transaction Draft
+        O ->> B: Started
+        B ->> O: Pending
+        O ->> Oc: Transaction Pending
+        B ->> Bc: Transaction Review
+    end
+
+    critical
+        Bc ->> Bc: KYC and Sanctions Checks
+    end
+
+    rect rgb(240,186,200)
+        note over O,B: RPC 2
+        Bc ->> B: Reject Transaction (no retry/repair)
+        B ->> O: Reject
+        O ->> B: Reject
+        B ->> Bc: Transaction Rejected
+        O ->> Oc: Transaction Rejected
+    end
+
+    critical
+        Oc ->> Oc: Stop on chain transaction
+    end
+```
+
+## Transaction States and Actions
+
+A transaction may be in one of the following states:
+
+| Status | Description |
+|---|---|
+| Unspecified | The transfer state is unknown or purposefully not specified (zero value) |
+| Draft | The transaction has been created but no secure envelopes have been sent yet |
+| Pending | Awaiting a response from the counterparty |
+| Review | Requires review by a compliance officer and some action taken |
+| Repair | An error has been sent back from the counterparty requesting a change |
+| Accepted | The TRISA exchange has been accepted by the counterparty; awaiting the on-chain transaction |
+| Completed | The TRISA exchange and the on-chain transaction have been completed |
+| Rejected | The TRISA exchange is rejected and no on-chain transaction should proceed |
+| Canceled | The TRISA exchange is canceled before it was completed |
+
+When a transaction is in one of the following states, a compliance officer or auto review policy must take one of the following actions:
+
+- **Draft**: Start an information exchange by _sending_ a secure envelope to the counterparty. This will cause the transaction to move to the Pending state.
```

Comment:
I noticed that there's an [`initialized` state](https://github.com/trisacrypto/envoy/blob/9db87528c0a2bab0b3762675c337d52dca490af0/pkg/store/sqlite/transactions.go#L418) for transactions that don't exist. This is likely a rare case, but should it be included in the documentation to prevent confusion?

Reply:
Ah - good catch; those strings should be converted into model state constants; I'll open a quick PR to fix that.

Reply:
Here is the PR with the fix: https://github.com/trisacrypto/envoy/pull/184